docs_source_files/content/support_packages/working_with_cookies.de.md

```diff
@@ -298,7 +405,35 @@ driver.add_cookie({"name": "test2", "value": "cookie2"})
 driver.delete_cookie("test1")
   {{< / code-panel >}}
   {{< code-panel language="csharp" >}}
-// Please raise a PR
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace DeleteCookie
+{
+    class DeleteCookie
+    {
+        public static void Main(string[] args) {
+            IWebDriver driver = new ChromeDriver();
+            try {
+                // Navigate to Url
+                driver.Navigate().GoToUrl("https://example.com");
+                driver.Manage().Cookies.AddCookie(new Cookie("test1", "cookie1"));
+                var cookie = new Cookie("test2", "cookie2");
+                driver.Manage().Cookies.AddCookie(cookie);
+
+		        // delete a cookie with name 'test1'	
```

Comment:
This looks way too indented to the right, could you please align it?

Reply:
Updated code as suggested